### ISSUE TITLE
make ARCHS an option in build_oc.tool

### ISSUE THREAD

**Issue by cattyhouse** [acidanthera/bugtracker#922]
## Issue one
build command:

`TARGETS=RELEASE RTARGETS=RELEASE ./build_oc.tool`

It builds ok, but there is an error message, does it matter?

````
...
Building OpenCorePkg/OpenCorePkg.dsc for X64 in RELEASE with XCODE5...
. - OK
./build_oc.tool: line 491: 72028 Terminated: 15          pingme $! build "$@"  (wd: ~/github/OpenCorePkg/UDK)
...

````

## Issue two

I'm able to pass `TARGETS=RELEASE RTARGETS=RELEASE`  to  `./build_oc.tool`, but not able to pass `ARCHS`, unless comment out `#ARCHS=(X64 IA32)`.

i'm building on macOS and i don't understand why IA32 was build as well. Can you make `ARCHS` an option? thanks

**Comment by kommtzeitkonrad** [comment]
> ## Issue one
> build command:
> 
> `TARGETS=RELEASE RTARGETS=RELEASE ./build_oc.tool`
> 
> It builds ok, but there is an error message, does it matter?
> 
> ```
> ...
> Building OpenCorePkg/OpenCorePkg.dsc for X64 in RELEASE with XCODE5...
> . - OK
> ./build_oc.tool: line 491: 72028 Terminated: 15          pingme $! build "$@"  (wd: ~/github/OpenCorePkg/UDK)
> ...
> ```
Ⅰ had similar issues, see closed https://github.com/acidanthera/bugtracker/issues/918.

**Comment by cattyhouse** [comment]
> > ## Issue one
> > build command:
> > `TARGETS=RELEASE RTARGETS=RELEASE ./build_oc.tool`
> > It builds ok, but there is an error message, does it matter?
> > ```
> > ...
> > Building OpenCorePkg/OpenCorePkg.dsc for X64 in RELEASE with XCODE5...
> > . - OK
> > ./build_oc.tool: line 491: 72028 Terminated: 15          pingme $! build "$@"  (wd: ~/github/OpenCorePkg/UDK)
> > ...
> > ```
> 
> Ⅰ had similar issues, see closed #918.

Thanks, i changed the title to refer Issue two.

**Comment by vit9696** [comment]
IA32 is because we may want to support 32-bit computers in some distant future and because we contribute code to EDK II, which supports different architectures. Fixed.
